src/algebra/free.lean

```diff
@@ -6,6 +6,7 @@ Authors: Kenny Lau
 import data.equiv.basic
 import control.applicative
 import control.traversable.basic
+import algebra.group.hom
 
 /-!
 # Free constructions
```

Comment:
Any chance you could add `free_magma.lift` to the list of main definitions? It doesn't need a very long description.